image/README.md

```diff
@@ -46,6 +46,11 @@ gcloud compute instances create $BUILD_MACHINE_NAME \
 gcloud compute firewall-rules create allow-ssh-ingress-from-iap --direction=INGRESS --action=allow --rules=tcp:22 --source-ranges=35.235.240.0/20 --network=$BUILD_MACHINE_NETWORK --project=$GOOGLE_CLOUD_PROJECT
 ```
 
+### Copy Knfsd Agent Source Code to Machine
+```
+gcloud beta compute scp --recurse ../knfsd-agent build@$BUILD_MACHINE_NAME: --zone=$BUILD_MACHINE_ZONE --tunnel-through-iap
```

Comment:
Should we also change the ssh step to connect with the `build` user for consistency, or remove `build@` from this step?

Also, should include `--project=$GOOGLE_CLOUD_PROJECT`.

Reply:
Going to keep this as is, the reason I used `build@` for this step is so that the Knfsd Agent files are in a predictable directory for the build steps to move but I think the actual SSH is better executed from the users actual username.

Reply:
I have added the `--project=$GOOGLE_CLOUD_PROJECT` flag.